src/main/java/com/bandwidth/sdk/exception/MissingCredentialsException.java

```diff
@@ -0,0 +1,26 @@
+package com.bandwidth.sdk.exception;
+
+import java.io.IOException;
+
+public class MissingCredentialsException extends IOException {
+
+    private static final long serialVersionUID = 8463765824822860908L;
+
+    public MissingCredentialsException() {
+        super("Missing credentials. \n" +
+                "\tThere are 3 ways to set these credentials:\n\n" +
+                "\t1. Via environment variables:\n" +
+                "\t\texport BANDWIDTH_USER_ID=<myUserId>\n" +
+                "\t\texport BANDWIDTH_API_TOKEN=<myApiToken>\n" +
+                "\t\texport BANDWIDTH_API_SECRET=<myApiSecret>\n" +
+                "\t2. Via Java VM System Properties, set as -D arguments on the VM command line:\n" +
```

Comment:
Change the order of the ways of setting the credentials. Place the System.properties first